pkg/testutil/testutil.go

```diff
@@ -55,21 +57,30 @@ func CreateContext(t *testing.T) (context.Context, *config.Config, func()) {
 
 	instanceTellor, err := tellor.NewTellor(contractAddress, client)
 	if err != nil {
-		log.Fatal(err)
+		t.Fatalf("Problem creating tellor master instance: %v\n", err)
 	}
 	ctx = context.WithValue(ctx, common.ContractsTellorContextKey, instanceTellor)
 
 	instanceGetter, err := getter.NewTellorGetters(contractAddress, client)
 	if err != nil {
-		log.Fatal("Problem creating tellor master instance", err)
+		t.Fatalf("Problem creating tellor master instance: %v\n", err)
 	}
 	ctx = context.WithValue(ctx, common.ContractsGetterContextKey, instanceGetter)
 
 	proxy, err := db.OpenLocalProxy(dbLocal)
 	if err != nil {
-		log.Fatal(err)
+		t.Fatalf("Problem creating proxy: %v\n", err)
 	}
 	ctx = context.WithValue(ctx, common.DataProxyKey, proxy)
 
 	return ctx, cfg, cleanup
 }
+
+func SetupLogger() log.Logger {
```

Comment:
instead of duplicating the func why not put it in `pkg/util/logger.go` and then can use for the tests and in the main entry point.